apps/admin-gui/src/app/core/services/common/admin-gui-config.service.ts

```diff
@@ -175,4 +177,13 @@ export class AdminGuiConfigService {
       }, error => reject(error));
     });
   }
+
+  private loadAppsConfig(): Promise<void>  {
+    return new Promise((resolve, reject) => {
+      this.utilsService.getAppsConfig().subscribe( appsConfig => {
+        this.store.setAppsConfig(appsConfig);
+        resolve();
+      }, error => reject(error));
+    });
+  }
```

Comment:
move this method to the "appConfigService" and also don't forget to call it also from user-profile-config.service!

Reply:
also call it from password reset and publications if you are using the forceRoute service

Reply:
Method has been moved and method call has been added to user-profile-config.service.

In publications and password reset applications we don't use links to other applications, so in my opinion, there it's not necessary.